src/angularjs/src/app/compare/compare.controller.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @ngdoc controller
+ * @name pfb.analysis-jobs.detail.controller:CompareController
+ *
+ * @description
+ * Controller for showing details about an analysis job
+ *
+ */
+(function() {
+    'use strict';
+
+    /** @ngInject */
+    function CompareController($stateParams, Neighborhood, AnalysisJob, $log, $location) {
+        var ctl = this;
+
+        initialize();
+
+        function initialize() {
+            ctl.places = [null, null, null];
+            ctl.getPlace = getPlace;
+            ctl.clearSelection = clearSelection;
+
+            getPlace(0, $stateParams.place1);
+            getPlace(1, $stateParams.place2);
+            getPlace(2, $stateParams.place3);
+        }
+
+        function getPlace(num, uuid) {
+            if (!uuid) {
+                return;
+            }
+
+            ctl.places[num] = {};
+
+            Neighborhood.query({uuid: uuid}).$promise.then(function(data) {
+                ctl.places[num].neighborhood = new Neighborhood(data);
```

Comment:
Hmm, I don't feel strongly about this one either way. I might lean slightly towards displaying all three at the same time for two reasons:
1. Each individual piece of content is less valuable until all the content is presented
2. Loading each block individually makes the page load as a whole feel a bit janky as content pops in piecemeal

If one of the blocks fails to load, you could still show the remaining blocks, and have some type of error block in place of the ones that failed to load. I think that would look the same in either case.

Not to punt down the chain, but @designmatty may have a preference as this feels more of a UI/UX issue.